system/Session/Session.php

```diff
@@ -35,17 +36,88 @@ class Session implements SessionInterface
      */
     protected $driver;
 
+    /**
+     * The session storage driver to use.
+     *
+     * @var string
+     */
+    protected $handler = FileHandler::class;
```

Comment:
I would be in favor of marking these new props as private to reduce BC concerns in the future. This also promotes encapsulation. If a user will find this implementation as lacking, they can abstract one using the `SessionInterface` instead of extending this and complaining this and that. But this is just my preference.

Reply:
I understand your preference.

But the way to extend the core classes is described in the user guide.
https://codeigniter4.github.io/CodeIgniter4/extending/core_classes.html#extending-core-classes
And CI users have been using it a lot for a long time.

Making properties as private is big change. I don't think it's easy to decide.